easybuild/framework/easyblock.py

```diff
@@ -1529,8 +1529,18 @@ def restore_iterate_opts(self):
 
     def det_iter_cnt(self):
         """Determine iteration count based on configure/build/install options that may be lists."""
-        iter_cnt = max([1] + [len(self.cfg[opt]) for opt in ITERATE_OPTIONS
-                              if isinstance(self.cfg[opt], (list, tuple))])
+        iter_opt_lens = [len(self.cfg[opt]) for opt in ITERATE_OPTIONS
```

Comment:
Not a big fan of this variable name, given the function name how about `iter_opt_counts`

Reply:
fixed in 23fd3a12c